### PR TITLE
Adds OpenShift to v3.0 dir as beta resumes support

### PR DESCRIPTION
**Pull request by emanic**
## Description

- Adds OpenShift install docs back into the v3.0 directory as we have established that it works.
- Also adjusts notes and intro pages accordingly.

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -33,7 +33,7 @@ clean:
 	docker run --rm -ti -e JEKYLL_UID=`id -u` -v $$PWD:/srv/jekyll jekyll/jekyll:$(JEKYLL_VERSION) jekyll clean
 
 htmlproofer: clean _site
-	docker run -ti -e JEKYLL_UID=`id -u` --rm -v $$PWD/_site:/_site/ quay.io/calico/htmlproofer /_site --file-ignore ${HP_IGNORE_LOCAL_DIRS} --assume-extension --check-html --empty-alt-ignore --url-ignore "/docs.openshift.org/,#,/github.com\/projectcalico\/calico\/releases\/download/"
+	docker run -ti -e JEKYLL_UID=`id -u` --rm -v $$PWD/_site:/_site/ quay.io/calico/htmlproofer /_site --file-ignore ${HP_IGNORE_LOCAL_DIRS} --assume-extension --check-html --empty-alt-ignore --url-ignore "/docs.openshift.org/,#,/github.com\/projectcalico\/calico\/releases\/download/,https://github.com/projectcalico/calico/tree/master/v3.0/getting-started/openshift/installation.md"
```
Comment (ozdanborne) [inline Makefile]: This needs to be backed out before merging, right?

Comment (ozdanborne) [reply Makefile]: Once this is resolved, LGTM

Comment (emanic) [reply Makefile]: YEs, I have to make new PR that removes the ignore, post merge

Comment (emanic) [reply Makefile]: Because I do not have the power to override this check. Unless you want to do it.

Comment (ozdanborne) [reply Makefile]: If lets take this out and then I can override the merge. We expect it to pass once in master, right?

Comment (emanic) [reply Makefile]: All right, I've pushed up this change. Yes, it is a chicken egg problem with the proofer. Sounds good 👍

**Pull request merged**